code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -890,6 +890,8 @@ func removeAllocationFromBlobber(
 	if err := blobAllocsParts.Save(balances); err != nil {
 		return fmt.Errorf("could not update blobber allocation partitions: %v", err)
 	}
+	// nullifying the blobber alloc challenge pasrtition location
+	allocPartLoc = nil
```

Comment:
This nullifying would not work as the allocPartLoc is a pointer param passed by the method caller, we can modify it's value through dereference (`*allocPartLoc`), but can not modify the pointer itself. So  perhaps we need to pass the blobberAllocation, and set its `BlobberAllocationsPartitionLoc` as nil.